src/full/Agda/Syntax/Translation/InternalToAbstract.hs

```diff
@@ -590,7 +592,8 @@ reifyTerm expandAnonDefs0 v0 = do
                      doDrop _         [] = []
                  in doDrop dropArg $ named_es'
 
-              simpl_named_es' | Just sub_mtel2local <- msub1 = dropIdentitySubs IdS           sub_mtel2local
+              simpl_named_es' | opt_show_ids                 = named_es'
```

Comment:
Who started to use snake_case? @Saizan Andrea! 39681e1c807c04d0556f9746576d66e741b9b21e
And this in the realm of camlCase!